torchvision/transforms/functional.py

```diff
@@ -1539,8 +1539,6 @@ def elastic_transform(
         fill (number or str or tuple): Pixel fill value for constant fill. Default is 0.
             If a tuple of length 3, it is used to fill R, G, B channels respectively.
             This value is only used when the padding_mode is constant.
-            Only number is supported for torch Tensor.
-            Only int or str or tuple value is supported for PIL Image.
```

Comment:
Drive-by since I was looking into the fill support. This seems to be a copy-paste error. Internally we just convert PIL images to tensors 

https://github.com/pytorch/vision/blob/59dc9383e663a9bab5230370e1f0d7d14b87940f/torchvision/transforms/functional.py#L1558-L1562

and then call the tensor kernel: 

https://github.com/pytorch/vision/blob/59dc9383e663a9bab5230370e1f0d7d14b87940f/torchvision/transforms/functional.py#L1573-L1578

Meaning, there is no difference between both types.